docs/UserGuide.md

```diff
@@ -261,28 +308,40 @@ Format: `find PREFIX KEYWORD [KEYWORD]…​`
 
 ### Adding a tag : `create-tag`
 
-Creates a `Tag` within WedLinker to be used on contacts.
+Use the create-tag command to create a `tag` within WedLinker, which you can assign to contacts.
 
 Format: `create-tag t/TAGNAME`
 
+To create a tag, type `tag` followed by the name of the tag.
```

Comment:
Yes, i'll fix that, but the second point i'm worried about commands like add command where theres multiple prefixes


> To add a contact to WedLinker, type `add` followed by details such as the name, phone number, and email.

it might become a tough to phrase it concisely